docs/user-guide/cli-using-global-profile-configuration.md

```diff
@@ -0,0 +1,535 @@
+
+### Benefits
+
+Global profiles improve your Zowe CLI experience:
+
+- As a user, you can manage your connection details efficiently in one location.
+- As a team leader, you can share global profiles with your team members so that they can easily access mainframe services. You can add the file directly to your project in an SCM.
+- As a new team member, you can onboard quickly by using your team's configuration file.
+
+### Changes to secure credential storage
+
+With the introduction of global profiles, the  Secure Credential Store (SCS) Plug-in is deprecated in the `@next` version. <!-- remove this plug-in from Brightside user docs --> The `zowe scs` and `zowe config` command groups are obsolete. Secure credential encryption is now included in the core CLI.
+
+The CLI prompts you to enter the username and password securely by default. Commands in the `zowe config` command group let you manage security for any option value. <!-- is this a new command group? the previous paragraph indicates that `zowe config` is deprecated -->
+
+
+## Initializing global configuration
+
+Verify that you have installed [the @next release of Zowe CLI](cli-install-cli-next.md).
+
+You have two options to initialize the global configuration:
+ * Define a connection to z/OSMF (recommended).
+ * Create your configuration files manually.
+
+***Follow these steps:***
+
+1. Issue the command:
+
+   ```
+   zowe config init --global-config
+   ```
+
+   The CLI responds with prompts for a username and password.
+
+2. Respond to the prompts with a username and password for a mainframe service such as z/OSMF.
+
+   **Note:** The `config init` command ensures that your credentials are stored securely on your computer by default.
+
+   After you respond, the `zowe.config.json` global configuration file is added to your local `.zowe` directory. This is the primary location where your MF service connection details such as host and port are defined. Use this configuration file for the following procedures.
+
+3. Edit the `zowe.config.json` file to edit the name of the LPAR,  supply a value for the host name, and edit the port number as needed.
+
+4. Issue a Zowe CLI command to test that you can access z/OSMF. For example, list all data sets under your user ID:
+
+   ```
+   zowe zos-files list data-set "MY.DATASET.*"
+   ```
+
+   A list of data sets is returned. You successfully configured Zowe CLI to access a z/OSMF instance.
+
+   If the CLI returns an error message, verify that you have access to the target system. Examine your configuration files in a text editor to verify that the information you entered is correct.
+
+**Important!:** After the configuration files are in place (either via the `zowe config init` command or by manually creating the files), the now-deprectaed `zowe profiles` commands will no longer function. Zowe CLI will return errors if you attempt to make use of old profiles.
```

Comment:
now-deprectaed -> now-deprecated

attempt to make use of old profiles -> now attempt to make use of the old profile commands.